WalletWasabi/Crypto/Strobe.cs

```diff
@@ -126,6 +127,11 @@ public override string ToString()
 			return ByteHelpers.ToHex(_state);
 		}
 
+		~Strobe128()
+		{
+			Array.Clear(_state, 0, _state.Length);
```

Comment:
good call! not spreading copies of secret state this is another reason to prefer the mutable approach, fwiw